CONTRIBUTING.md

```diff
@@ -15,35 +24,42 @@ Did you find a bug in GeoBlacklight or interested in a new feature? Make sure to
     - Explain the actual behavior
     - Provide steps to reproduce the actual behavior
 
-### GeoBlacklight Software Versioning
-GeoBlacklight follows the practice of [Semantic Versioning](https://semver.org/) for software releases. The declared semantically versioned public API includes:
- - the GeoBlacklight-Schema
- - the [public GeoBlacklight Ruby codebase classes](http://www.rubydoc.info/github/geoblacklight/geoblacklight/master/frames)
- - the GeoBlacklight JavaScript interface
- - the GeoBlacklight view interface
+### Issue Assignment
+Issues are used in the GeoBlacklight community as a convenient way to document bugs, propose new functionality and enhancements, and discuss important code changes.
+
+- If someone is already assigned to an issue, that does not mean you can't contribute.
```

Comment:
Do contributors need to assign issues? Can they assign themselves?

Reply:
I don't think the community has that expectation. I think assignment is used informally during sprints to help dish out issues for folks to work on. Maybe we want to say some version of that?

In some communities people will self assign to indicate they are working on it. This section could probably use more feedback!

We don't want people thinking they can't share ideas because an issue is "claimed", but if someone's working on something and self-assign, we may not want them to expect that nobody else will chime in.

Reply:
@karenmajewicz @ewlarson thoughts on this?

Reply:
How about this?

### Issues

{{create a header with the general description of issues}}

Issues are used in the GeoBlacklight community as a convenient way to document bugs, propose new functionality and enhancements, and discuss important code changes.

#### Reporting Issues

{{same}}

Did you find a bug in GeoBlacklight or interested in a new feature? Make sure to add an issue for it in the [issue tracker](https://github.com/geoblacklight/geoblacklight/issues).

 - Make sure you have a [GitHub account](https://github.com/signup/free)
 - Submit a [GitHub issue](./issues) by:
    - Clearly describing the issue
    - Provide a descriptive summary
    - Explain the expected behavior
    - Explain the actual behavior
    - Provide steps to reproduce the actual behavior

#### Assigning Issues

{{a little more explanation}}

In the GeoBlacklight community, issue assignment helps signal who is interested in working on an issue or who can assist in moving it forward. If you find an issue you’d like to contribute to, you can assign yourself to it. If an issue is already assigned, you are still welcome to contribute or provide support. Active issues will be marked with a status of “In progress” on our Project boards.

Reply:
Updated!